aten/src/ATen/native/cuda/Loops.cuh

```diff
@@ -94,24 +95,36 @@ Loops.cuh (and CUDALoops.cuh) usually build. This reduces build time,
 build size, and CUDA context size.
 
 The jiterator currently has some limitations, however. It cannot:
-  - handle float16, bfloat16, or complex datatypes
-  - handle scalar inputs
+  - handle complex datatypes
```

Comment:
Thank you for updating this comment. More precisely it cannot handle math on complex datatypes (we now support converting to and from complex datatypes).